src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -18,11 +18,6 @@ using namespace Microsoft::Console::Render;
 using namespace Microsoft::Console::Types;

```

Comment:
nit: I'd probably drop a 
```suggestion
using namespace Microsoft::Console::Types;
using namespace Microsoft::Console;
```
here

Reply:
ah, I was copying the references in `Terminal::_InitializeColorTable`. I'll change those too